src/js/view/utils/views-manager.js

```diff
@@ -56,11 +77,19 @@ document.addEventListener('webkitvisibilitychange', onVisibilityChange);
 window.addEventListener('resize', scheduleResponsiveRedraw);
 window.addEventListener('orientationchange', scheduleResponsiveRedraw);
 
+if (OS.android && Browser.chrome) {
+    window.screen.orientation.addEventListener('change', onOrientationChange);
```

Comment:
`OS.android && Browser.chrome` Does not guarantee that `window.screen.orientation` is defined. 

`'screen' in window && 'orientation' in window.screen`

Reply:
I didn't say remove the Android Chrome check. We still need to gate this feature. The point is that you still need to guard from writing JS that can throw exceptions.